packages/yoga/src/Progress/native/Progress.jsx

```diff
@@ -116,7 +118,7 @@ const Label = styled.Text`
  * of quantity.  The use of labels numeric or alphabetic can increase the user
  * understanding. */
 const Progress = ({ label, max, value, variant, ...props }) => {
-  const isNumber = !isNaN(label.value);
+  const isNumber = !/[a-zA-Z]/g.test(label.value);
```

Comment:
With this regex it will allow especial characters like `ç, é, ó`, maybe it's better uses `\D` to get everything what aren't a digit.